Summary of balancing options

greg-szrama opened the pull request:
Resolves https://github.com/redpanda-data/documentation-private/issues/1847

Added table summarizing balancing options

[Preview.](https://deploy-preview-359--redpanda-docs-preview.netlify.app/current/manage/cluster-maintenance/cluster-balancing/)

hcoyote commented:
Formatting is wrapping a little odd in the middle column for me, on Chrome.

<img width="1193" alt="image" src="https://github.com/redpanda-data/docs/assets/751025/0320429c-2894-4786-8e3b-18038b402558">

hcoyote commented on modules/manage/pages/cluster-maintenance/cluster-balancing.adoc:
```diff
@@ -9,6 +9,30 @@ With an Enterprise license, you can additionally enable _Continuous Data Balanci
+|===
+|Balancer |License |Description
+
+|Leadership balancing
```
Can we unify the balancer names used here in the table with what's later in the topic?

So, Leadership Balancing -> Partition Leadership Balancing

hcoyote commented on modules/manage/pages/cluster-maintenance/cluster-balancing.adoc:
```diff
@@ -9,6 +9,30 @@ With an Enterprise license, you can additionally enable _Continuous Data Balanci
+Enterprise
+|Topic-aware balancer designed to distribute partition leadership across available nodes. This helps avoid topic leadership hotspots on one or a few specific nodes in your cluster.
+
+|Partition replica balancing
```
Partition Replica Balancing is probably a clearer name than what we have below (Redpanda Partition Balancing ) ... might be worth changing the section name to be Partition Replica Balancing?

kbatuigas commented on modules/manage/pages/cluster-maintenance/cluster-balancing.adoc:
```diff
@@ -17,7 +41,7 @@ Leadership balancing is enabled by default with the xref:reference:cluster-prope
 The xref:reference:cluster-properties.adoc#leader_balancer_mode[`leader_balancer_mode`] property ensures that each shard in a cluster has an equal number of partitions. It determines the movement of leadership for the replica set of a partition. It supports two modes:
 
-* `random_hill_climbing`: This mode randomly searches for potential leadership movements. If one is found that better balances the number of leaders per shard and the leaders of a given topic per broker, then the movement is applied to the cluster. This is the default.
+* `random_hill_climbing`: This mode randomly searches for potential leadership movements. If one is found that better balances the number of leaders per shard and the leaders of a given topic per broker, then the movement is applied to the cluster4. This is the default.
```
```suggestion
* `random_hill_climbing`: This mode randomly searches for potential leadership movements. If one is found that better balances the number of leaders per shard and the leaders of a given topic per broker, then the movement is applied to the cluster. This is the default.
```
Typo?

kbatuigas reviewed (approved):
Spotted on possible typo? But lgtm otherwise!

micheleRP commented on modules/manage/pages/cluster-maintenance/cluster-balancing.adoc:
```diff
@@ -9,6 +9,30 @@ With an Enterprise license, you can additionally enable _Continuous Data Balanci
+Enterprise
+|Topic-aware balancer designed to distribute partition replicas when adding nodes to the cluster. This helps avoid topic replica hotspots on one or a few specific nodes in your cluster.
+
+|Continuous data balancing
```
```suggestion
|Continuous Data Balancing
```
@greg-szrama Continuous Data Balancing is capitalized because it is an enterprise feature

Feediver1 commented:
@greg-szrama The link to **Licenses** in the table is not working.

Feediver1 commented on modules/manage/pages/cluster-maintenance/cluster-balancing.adoc:
```diff
@@ -9,6 +9,28 @@ With an Enterprise license, you can additionally enable _Continuous Data Balanci
+.Cluster balancing options
+[cols="4,2,11"]
+|===
+|Balancer |https://deploy-preview-359--redpanda-docs-preview.netlify.app/current/get-started/licenses/[License] |Description
```
```suggestion
|https://deploy-preview-359--redpanda-docs-preview.netlify.app/current/get-started/licenses/[License] |Description
```

greg-szrama commented:
Fixed that link, as well as the link to 'Configure Continuous Data Balancing' which was using a relative path too. I also fixed a minor grammar thing (self-heals instead of self heals). Merging!
